tests/TestCase.php

```diff
@@ -31,4 +33,13 @@ protected function defineEnvironment($app)
             // 'query-builder.request_data_source' => 'body',
         ]);
     }
+
+    public static function cachedConfigurationForWorkbench(): Config
+    {
+        $config = Workbench::configuration();
+
+        $config['seeders'] = false;
+
+        return $config;
+    }
```

Comment:
Can you explain why you need to add this.

Reply:
Test seeders configured in the workbench config were running in the test suite. I think as a result of the `TestCase` class using the `WithWorkbench` trait. 

I've not done a lot of digging, but this fixed the issue. If the issue is deeper (or even a bug in testbench-core).

Reply:
`TestCase` wouldn't run `seeders` unless the `$seed` property is set to `true`. I can't find anywhere this is set for this repo.

https://github.com/laravel/framework/blob/2747a5ffe0d4dcdcc39b0cd3b3dd9892b619c0c4/src/Illuminate/Foundation/Testing/Traits/CanConfigureMigrationCommands.php#L50-L53

Reply:
Also, https://github.com/cachethq/core/pull/96/commits/baf12e52fccfd909d2856961a0be51001bb36288 would limit seeders to only be executed via `workbench:build` command.

Reply:
@crynobone i don't know what you're looking for from me. Try to pull this down and run the test suite with this removed. The seeders run and the tests that assert database counts exist fail 🤷

Reply:
Sorry, wanting to understand the issue as I maintain Testbench and curious to see what we can improves

Reply:
I appreciate that, thank you!

I did try to do some digging and found that the problem exists in the configuration returned from `Workbench::configuration()` [here](https://github.com/orchestral/testbench-core/blob/60dd2ad451cfcf3e85fcf15ff43d920b0f263ed3/src/Concerns/WithWorkbench.php#L27)